src/Runner.Worker/ExecutionContext.cs

```diff
@@ -1012,6 +1031,7 @@ public static class WellKnownTags
         public static readonly string Command = "##[command]";
         public static readonly string Error = "##[error]";
         public static readonly string Warning = "##[warning]";
+        public static readonly string Notice = "##[notice]";
```

Comment:
we might want to make the UI changes first before we start writing this.... actually i guess it doesnt matter because no one will be using this initially. So we have some time to make that change in parallel